core/pom.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.jenkins.plugin-modernizer</groupId>
+        <artifactId>plugin-modernizer-parent-pom</artifactId>
+        <version>${revision}${changelist}</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>core</artifactId>
```

Comment:
I think using a suffix is more clear for the artifact ID of child module

`plugin-modernizer-parent-core`
`plugin-modernizer-parent-cli`

Reply:
is `plugin-modernizer-core, plugin-modernizer-cli, plugin-modernizer-recipes, plugin-modernizer-integration` ok? (or) `parent` should be added to each module?

Reply:
Looks good.  `parent` (or `pom`) is only for the top level pom module